bbb_presentation_video/renderer/tldraw/utils.py

```diff
@@ -138,6 +190,8 @@ class FontStyle(Enum):
     FontStyle.ERIF: "Crimson Pro",
     FontStyle.SERIF: "Crimson Pro",
     FontStyle.MONO: "Source Code Pro",
+    FontStyle.DRAW: "Caveat Brush",
+    FontStyle.ARIAL: "Arial 12px",
```

Comment:
This doesn't seem right - why is the font size in the face name here? This seems like it will cause problems.

Arial font is ok - we were already using Arial for the pre-tldraw whiteboard, so we have a compatible font (actually [Arimo](https://fonts.google.com/specimen/Arimo)) pulled in via a dependency on fonts-croscore.